Documentation/network/bgp-control-plane/bgp-control-plane.rst

```diff
@@ -82,5 +83,4 @@ Troubleshooting and Operation Guide
     :glob:
 
     bgp-control-plane-troubleshooting
-    bgp-control-plane-operation
-
+    bgp-control-plane-operation
```

Comment:
This change in the second commit is to cancel the change from the first commit. Please fix in the first commit, not in the second!